utils/utils.ts

```diff
@@ -129,7 +129,7 @@ export const getUserLeagues = async (
     return [];
   }
   const userLeagues = leagues.map((league) => {
```

Comment:
please revert this back to what it was.  It should be taking in an array of league id's and then all those are fetched with the getCurrentLeague function using the promise.all.

Reply:
was this updated per what I had asked?